templates/security/cis.html

```diff
@@ -37,38 +37,31 @@ <h2 class="p-heading--4">Comply with the most widely accepted Linux baseline</h2
     <div class="col-4 p-divider__block">
       <h3>Harden your Linux workloads</h3>
       <p>Hardening involves a tradeoff between security and usability. The default configuration of Ubuntu LTS releases, as provided by Canonical, balances between usability, performance and security. However, systems with a dedicated workload are well-positioned to benefit from hardening. Reduce your Linux workload’s attack surface with CIS hardened Ubuntu.</p>
-      <a href="/blog/cis-hardened-ubuntu-cyber-attack-and-malware-prevention-for-mission-critical-systems" class="p-button">Read more about Ubuntu CIS</a>
     </div>
     <div class="col-4 p-divider__block">
       <h3>Automate your compliance</h3>
-      <p>Applying a baseline with a large set of instructions manually is not only time consuming but also error-prone. According to <a class="p-link--external" href="https://enterprise.verizon.com/resources/reports/dbir/">Verizon data breach investigations</a> report for 2021, misconfigurations were among the top five reasons for data breaches. Apply more than 250 rules in less than 15 minutes while avoiding misconfigurations using tooling provided by Canonical that automates your CIS compliance.</p>
-      <a href="/security/certifications/docs/cis" class="p-button">Read more about Ubuntu’s tooling</a>
+      <p>Applying a baseline with a large set of instructions manually is not only time consuming but also error-prone. According to <a href="https://enterprise.verizon.com/resources/reports/dbir/">Verizon data breach investigations</a> report for 2021, misconfigurations were among the top five reasons for data breaches. Apply more than 250 rules in less than 15 minutes while avoiding misconfigurations using Ubuntu Security Guide (USG) that automates your CIS compliance.</p>
```

Comment:
I've left a comment for Nikos in the copy doc, so let's wait for a response from him first, but I'm not sure we need this abbreviation, it isn't used anywhere else on the page, whereas the full "Ubuntu Security Guide" is used in several:
```suggestion
      <p>Applying a baseline with a large set of instructions manually is not only time consuming but also error-prone. According to <a href="https://enterprise.verizon.com/resources/reports/dbir/">Verizon data breach investigations</a> report for 2021, misconfigurations were among the top five reasons for data breaches. Apply more than 250 rules in less than 15 minutes while avoiding misconfigurations using Ubuntu Security Guide that automates your CIS compliance.</p>
```